infrastructure/kube/keep-test/monitoring/configmap.yaml

```diff
@@ -0,0 +1,656 @@
+---
```

Comment:
Maybe we could split this file into: `prometheus-configmap.yaml` and `grafana-configmap.yaml`?